### PR TITLE
Add ticket templates for kubedb databases

### PR DESCRIPTION
**Pull request by heheh13**

### CODE REVIEW

File: charts/mysql/ticket-template.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+This template is designed to contact with AppsCode with troubleshooting and support in general. You will find all the KubeDB Managed database follows similar pattern. To troubleshoot and find what to do let's get started with the database phase.
+
+Let's say you have MySQL database in namespace demo.Please refer to [KubeDB Docs](https://kubedb.com/docs/v2023.01.31/guides/mysql/) for more about KubeDB.
```
Comment (Shaad7) [inline charts/mysql/ticket-template.md:3]: Is this link static to release v2023.01.31? If yes, it should be dynamic or we can just forward it to kubedb.com/docs

File: charts/mysql/ticket-template.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+```bash
+Kubectl get mysql <ns>     // wiil list all the database in a namesapce
```
Comment (Shaad7) [inline charts/mysql/ticket-template.md:5]: kubectl get mysql -n <ns>

File: charts/mysql/ticket-template.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Let's say you have MySQL database in namespace demo.Please refer to [KubeDB Docs](https://kubedb.com/docs/v2023.01.31/guides/mysql/) for more about KubeDB.
+```bash
+Kubectl get mysql <ns>     // wiil list all the database in a namesapce  
+$ kubectl get mysql -n demo
+  NAME    VERSION   STATUS   AGE
+  mysql   8.0.31    Ready    6h51m
+```
+There are four different db phase you may see in KubeDB managed Database.
+``Ready`` ``Provisioning`` ``critical`` ``NotReady``
```
Comment (Shaad7) [inline charts/mysql/ticket-template.md:11]: critical -> Critical

File: charts/mysql/ticket-template.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+You can contact to AppsCode with the following things attached,
+- Get the MySQL object:
+    ```bash
+    kubectl get mysql -n <ns> <mysql-object-name> -oyaml
+    Kubectl get pod -n <demo> <pod-name> -oyaml 
+    ```
+- Describe the MySQL object: 
+    ```bash
+      kubectl describe mysql -n <ns> <mysql-ojbect-Name>
```
Comment (Shaad7) [inline charts/mysql/ticket-template.md:27]: typoe in object

**Review by Shaad7 — CHANGES_REQUESTED** [review]

File: charts/mysql/ticket-template.md (hunk 1 above)
Comment (heheh13) [reply charts/mysql/ticket-template.md:3]: I'm not aware how to do it . any idea?

Comment (Shaad7) [reply charts/mysql/ticket-template.md:3]: https://kubedb.com/docs/latest/guides/redis/ seems to work

File: charts/mysql/ticket-template.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+This template is designed to contact with AppsCode with troubleshooting and support in general. You will find all the KubeDB Managed database follows similar pattern. To troubleshoot and find what to do let's get started with the database phase.
+
+Let's say you have MySQL database in namespace demo.Please refer to [KubeDB Docs](https://kubedb.com/docs/v2023.01.31/guides/mysql/) for more about KubeDB.
+```bash
+Kubectl get mysql -n <ns>     // wiil list all the database in a namesapce
```
Comment (faem) [inline charts/mysql/ticket-template.md:5]: will

File: charts/mysql/ticket-template.md (hunk 1 above)
Comment (heheh13) [reply charts/mysql/ticket-template.md:3]: @faem vai

Comment (faem) [reply charts/mysql/ticket-template.md:3]: yap. use `latest` instead of specific version.

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]
@all 
Check the secret's existence by getting the secret name first. Like below : 
```bash
# Get the secret name
kubectl get mg -n <namespace> <mongodb-object-Name> -o jsonpath="{.spec.authSecret.name}"
# Check secret's existence
kubectl get secrets -n <namespace> <mongodb-secret-name>
```
Because externally managed secrets can be of different name

**Pull request merged**